recipes/behst/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.6" %}
+
+package:
+  name: behst
+  version: {{ version }}
+
+source:
+  fn: behst-{{ version }}.tar.gz
+  url: https://bitbucket.org/hoffmanlab/behst/get/tip.tar.bz2
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - pandas
+    - bedtools
+    - pybedtools
+    - r-base
+    - r-rcurl
+    - r-gprofiler
+    - gnu-wget
+
+test:
+    commands:
+      - behst.py -h
```

Comment:
other commands can be added here, but `run_test.sh` below should check everything